components/filter/test/filter-tags.test.js

```diff
@@ -0,0 +1,227 @@
+					await waitUntil(() => updateSpy.callCount === 2, 'Active filters were not updated a second time');
+
+					const filters = filterTags.shadowRoot.querySelectorAll('d2l-tag-list-item');
+					expect(filters.length).to.equal(0);
+					const updatedTagList = filterTags.shadowRoot.querySelector('d2l-tag-list');
+					expect(updatedTagList).to.be.null;
+				});
+			});
+		});
+	});
+
+	describe('Multiple filters', () => {
+		let filter1, filterTags;
+
+		beforeEach(async() => {
+			const elem = await fixture(twoFilters);
+			filter1 = elem.querySelector('#filter-1');
+			filterTags = elem.querySelector('d2l-filter-tags');
+			await filter1.updateComplete;
+			await filterTags.updateComplete;
+			const activeMap = filterTags._allActiveFilters;
+			await waitUntil(() => activeMap.get('filter-1').length === 1 && activeMap.get('filter-2').length === 1, 'Active filters were not set');
+
+			const tagList = filterTags.shadowRoot.querySelector('d2l-tag-list');
+			await waitUntil(() => tagList._items, 'Tag list items did not become ready');
+		});
+
+		it('instantiates with the selected filters added as multi-select items', async() => {
+			expect(filterTags.filterIds).to.equal('filter-1 filter-2');
+			const items = filterTags.shadowRoot.querySelectorAll('d2l-tag-list-item');
+			expect(items.length).to.equal(2);
+			expect(items[0].text).to.equal('Category 1: Value 1 - 1');
+			expect(items[1].text).to.equal('Value');
+		});
+		it('selecting another filter value adds to the list', async() => {
+			const updateSpy = spy(filterTags, 'updateActiveFilters');
+			const values = filter1.querySelectorAll('d2l-filter-dimension-set-value');
+			values[1].selected = true;
+			await waitUntil(() => updateSpy.callCount > 0, 'Active filters were not updated');
+
+			expect(filterTags._allActiveFilters.get('filter-1').length).to.equal(2);
+			expect(filterTags._allActiveFilters.get('filter-2').length).to.equal(1);
+			const items = filterTags.shadowRoot.querySelectorAll('d2l-tag-list-item');
+			expect(items.length).to.equal(3);
+			expect(items[0].text).to.equal('Category 1: Value 1 - 1');
+			expect(items[1].text).to.equal('Category 2: Value 2 - 1');
+			expect(items[2].text).to.equal('Value');
+		});
+		it('removing a filter value removes from the list', async() => {
+			const updateSpy = spy(filterTags, 'updateActiveFilters');
+			const values = filter1.querySelectorAll('d2l-filter-dimension-set-value');
+			values[0].selected = false;
+			await waitUntil(() => updateSpy.callCount > 0, 'Active filters were not updated');
+
+			expect(filterTags._allActiveFilters.get('filter-1')).to.be.empty;
+			expect(filterTags._allActiveFilters.get('filter-2').length).to.equal(1);
+			const items = filterTags.shadowRoot.querySelectorAll('d2l-tag-list-item');
+			expect(items.length).to.equal(1);
+			expect(items[0].text).to.equal('Value');
+		});
+		it('deleting an item from the multi-select list deselects the corresponding value from the filter', async() => {
```

Comment:
```suggestion
		it('deleting an item from the tag-list deselects the corresponding value from the filter', async() => {
```

Reply:
Thanks for catching all these - will fix!